From stable-bounces@linux.kernel.org Mon Feb 12 00:53:56 2007
From: Jiri Slaby <jirislaby@gmail.com>
Date: Mon, 12 Feb 2007 00:52:30 -0800
Subject: Char: specialix, isr have 2 params
To: torvalds@linux-foundation.org
Cc: akpm@linux-foundation.org, jirislaby@gmail.com, stable@kernel.org
Message-ID: <200702120852.l1C8qUKK026605@shell0.pdx.osdl.net>


From: Jiri Slaby <jirislaby@gmail.com>

specialix, isr have 2 params

pt_regs are no longer the third parameter of isr, call sx_interrupt without
it.

Signed-off-by: Jiri Slaby <jirislaby@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/char/specialix.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- linux-2.6.20.1.orig/drivers/char/specialix.c
+++ linux-2.6.20.1/drivers/char/specialix.c
@@ -459,7 +459,7 @@ void missed_irq (unsigned long data)
 	if (irq) {
 		printk (KERN_INFO "Missed interrupt... Calling int from timer. \n");
 		sx_interrupt (((struct specialix_board *)data)->irq,
-		              (void*)data, NULL);
+				(void*)data);
 	}
 	missed_irq_timer.expires = jiffies + sx_poll;
 	add_timer (&missed_irq_timer);
